openshift/topics/advanced_concepts.adoc

```diff
@@ -405,6 +405,68 @@ When restarting the server it is important to restart just the JBoss EAP process
 ====
 . Log in to the `master` realm's administration console of the {project_name} server using the credentials created in the steps above. In the browser, navigate to *\http://sso-<project-name>.<hostname>/auth/admin*  for the {project_name} web server, or to *\https://secure-sso-<project-name>.<hostname>/auth/admin* for the encrypted {project_name} web server, and specify the user name and password used to create the administrator user.
 
+
+=== Customizing the default behavior of the {project_name} image
+
+You can change the default behavior of the {project_name} image such as enabling TechPreview features or enabling debugging. This section describes how to make this change by using the JAVA_OPTS_APPEND variable.
+
+.Prerequisites 
+
+This procedure assumes that the {project_openshift_product_name} image has been previously link:{project_doc_base_url}/red_hat_single_sign-on_for_openshift/index#Example-Deploying-SSO[deployed using one of the following templates:]
+
+* *_{project_templates_version}-mysql_*
+* *_{project_templates_version}-mysql-persistent_*
+* *_{project_templates_version}-postgresql_*
+* *_{project_templates_version}-postgresql-persistent_*
+* *_{project_templates_version}-x509-mysql-persistent_*
+* *_{project_templates_version}-x509-postgresql-persistent_*
+
+.Procedure
+
+You can use the OpenShift web console or the CLI to change the default behavior.
+
+If you use the OpenShift web console, you add the JAVA_OPTS_APPEND variable to the sso deployment config. For example, to enable TechPreview features, you set the variable as follows:
+
+[source,bash,subs=+attributes]
+----
+JAVA_OPTS_APPEND="-Dkeycloak.profile=preview"
+----
+
+If you use the CLI, use the following commands to enable TechPreview features when the {project_name} pod was deployed using a template that is mentioned under Prerequisites.
+
+. Scale down the {project_name} pod:
+
++
+[source,bash,subs=+attributes]
+----
+$ oc get dc -o name
+deploymentconfig/sso
+deploymentconfig/sso73-mysql
```

Comment:
This should be changed back to the original  _"deploymentconfig/sso-mysql"_  form (see e.g.:

- https://github.com/jboss-container-images/redhat-sso-7-openshift-image/blob/sso73-dev/templates/sso73-mysql-persistent.json#L706
- https://github.com/jboss-container-images/redhat-sso-7-openshift-image/blob/sso73-dev/templates/sso73-mysql-persistent.json#L27)

What I wanted to say, when reviewing [previous version](https://issues.jboss.org/browse/KEYCLOAK-7010?focusedCommentId=13803054&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13803054) is, that "sso-mysql" template doesn't exist (it's not a template, but rather deploymentconfig). Didn't mean, the row above to be changed to "sso73-mysql"